packages/modals/src/styled/StyledDrawerModalHeader.spec.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render, renderRtl, screen } from 'garden-test-utils';
+import { DEFAULT_THEME, PALETTE } from '@zendeskgarden/react-theming';
+
+import { StyledDrawerModalHeader } from './StyledDrawerModalHeader';
+
+describe('StyledDrawerModalHeader', () => {
+  it('renders default styling', () => {
+    const { container } = render(<StyledDrawerModalHeader />);
+
+    expect(container.firstChild).toHaveStyleRule('color', DEFAULT_THEME.colors.foreground);
+  });
+
+  it('renders danger styling if provided', () => {
+    const { container } = render(<StyledDrawerModalHeader isDanger />);
+
+    expect(container.firstChild).toHaveStyleRule('color', PALETTE.red[600]);
+  });
+
+  it('renders correctly when button is present', () => {
```

Comment:
This is already covered at the element spec level: `has horizontal padding when DrawerModal.Close is present`.

Reply:
Removed these in [a6cf3f8](https://github.com/zendeskgarden/react-components/pull/1307/commits/a6cf3f8251f90ab9df1452314f08e94632f9fce8)!